sync/class.jetpack-sync-actions.php

```diff
@@ -87,7 +87,8 @@ static function add_sender_shutdown() {
 
 	static function sync_allowed() {
 		require_once dirname( __FILE__ ) . '/class.jetpack-sync-settings.php';
-		return ( ! Jetpack_Sync_Settings::get_setting( 'disable' )
+		return ( ( ! Jetpack_Sync_Settings::get_setting( 'disable' )
```

Comment:
Since checking whether Jetpack is disabled is now a compound conditional, I think we should add an `is_disabled()` method to `Jetpack_Sync_Settings` that will return a single boolean after evaluating the settings.